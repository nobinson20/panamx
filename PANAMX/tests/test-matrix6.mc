int main() {
    matrix m;
    matrix n;
    matrix q;
    m = [1, 9, 3;
         2, 5, 7;
         3, 6, 4];
    n = [17.1, 23.2, 10.9;
         15.6, 21.7, 10.5;
         9.8,  18.2, 21.5];
    q = m + n;
    printm(q);
    q = n - m;
    printm(q);
    return 0;
}
