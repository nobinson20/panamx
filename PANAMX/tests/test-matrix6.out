[
	18.100   32.200   13.900   
	17.600   26.700   17.500   
	12.800   24.200   25.500   
]
[
	16.100   14.200   7.900   
	13.600   16.700   3.500   
	6.800   12.200   17.500   
]
